HHS/OPRE-OPS
# feat: ✨adds pagination to Can Budget Lines

## What changed

- adds pagination to the budget lines table on the CAN Spending Tab

## Issue

#2672 #3053

## How to test

1. goto cans list
2. select fy2021
3. select CAN 4426 from table
4. goto spending tab
5. select fy2043
6. see table pagination // 3 per page for now

## Screenshots

<img width="1019" alt="image" src="https://github.com/user-attachments/assets/034bc726-9f10-4a8b-9ea3-03d51b84050f">

## Definition of Done Checklist
